Update/some_004.txt

```diff
@@ -2631,7 +2631,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c8647c>暁</color>", NULL, "<color=#c8647c>Akira</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 39, "ps3/s04/39/310200015", 256, TRUE);
 	OutputLine(NULL, "「————」",
-			NULL, "\"...\"", Line_Normal);
+			NULL, "\"————\"", Line_Normal);
```

Comment:
This would be a pretty far-reaching change, since the stand-alone ellipsis is used a TON in the console arcs. Since it's not really part of the translated Higurashi style, I'm not sure I'd recommend this.

Reply:
Yeah, we'll revert this. Using ellipsis is consistent throughout Higurashi - both official arcs and console arcs so we should keep it that way.

Reply:
```suggestion
			NULL, "\"...\"", Line_Normal);
```